spectacles/logger.py

```diff
@@ -57,26 +56,21 @@ def format(self, record):
 
 
 def log_sql_error(
-    error: SqlError,
-    log_dir: str,
-    model_name: str,
-    explore_name: str,
-    dimension_name: Optional[str] = None,
+    model: str, explore: str, sql: str, log_dir: str, dimension: Optional[str] = None
 ) -> Path:
-
     file_name = (
-        model_name
+        model
         + "__"
-        + explore_name
-        + ("__" + dimension_name if dimension_name else "")
+        + explore
+        + ("__" + dimension.replace(".", "_") if dimension else "")
```

Comment:
Do we need to do this `replace()` on the model and explore? Can they have `.` in them?

Reply:
Yeah, remember how we had to change the selector syntax to `model/explore` because someone apparently put dots in their names?

Reply:
Good idea.